seacrowd/sea_datasets/ud/ud.py

```diff
@@ -0,0 +1,250 @@
+            "id_csui": "UD_Indonesian-CSUI", 
+            "id_pud" : "UD_Indonesian-PUD", 
+            "vi_vtb": "UD_Vietnamese-VTB",
+            "tl_trg": "UD_Tagalog-TRG",
+            "tl_ugnayan": "UD_Tagalog-Ugnayan"
+            }
+
+_DESCRIPTION = """\
+Universal Dependencies (UD) is a project that is developing cross-linguistically consistent treebank annotation
+ for many languages, with the goal of facilitating multilingual parser development, cross-lingual learning, and 
+ parsing research from a language typology perspective. The annotation scheme is based on an evolution of (universal)
+   Stanford dependencies (de Marneffe et al., 2006, 2008, 2014), Google universal part-of-speech tags 
+   (Petrov et al., 2012), and the Interset interlingua for morphosyntactic tagsets (Zeman, 2008). 
+   The general philosophy is to provide a universal inventory of categories and guidelines to facilitate consistent
+     annotation of similar constructions across languages, while allowing language-specific extensions when necessary.
+"""
+
+_HOMEPAGE = "https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-5287"
+
+_LICENSE = ["Apache license 2.0 (apache-2.0)"]
+
+_URLS = {
+    "ud-v2.12": "https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11234/1-5150/ud-treebanks-v2.12.tgz?sequence=1&isAllowed=y",
+    "ud-v2.13": "https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11234/1-5287/ud-treebanks-v2.13.tgz?sequence=1&isAllowed=y",
+    "id_csui": {
+        "train": "https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-CSUI/master/id_csui-ud-train.conllu",
+        "test": "https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-CSUI/master/id_csui-ud-test.conllu",
+    },
+    "id_gsd": {
+        "train": "https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-GSD/master/id_gsd-ud-train.conllu",
+        "test": "https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-GSD/master/id_gsd-ud-test.conllu",
+        "dev": "https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-GSD/master/id_gsd-ud-dev.conllu",
+    },
+    "id_pud": {
+        "test": "https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-PUD/master/id_pud-ud-test.conllu"
+    },
+    "vi_vtb": {
+        "train": "https://raw.githubusercontent.com/UniversalDependencies/UD_Vietnamese-VTB/master/vi_vtb-ud-train.conllu",
+        "test": "https://raw.githubusercontent.com/UniversalDependencies/UD_Vietnamese-VTB/master/vi_vtb-ud-test.conllu",
+        "dev": "https://raw.githubusercontent.com/UniversalDependencies/UD_Vietnamese-VTB/master/vi_vtb-ud-dev.conllu",
+    },
+    "tl_trg": {
+        "test": "https://github.com/UniversalDependencies/UD_Tagalog-TRG/blob/master/tl_trg-ud-test.conllu",
+    },
+    "tl_ugnayan": {
+        "test": "https://github.com/UniversalDependencies/UD_Tagalog-Ugnayan/blob/master/tl_ugnayan-ud-test.conllu",
+    },
```

Comment:
changing these makes the file loaded properly for `tgl` languages:
```suggestion
    "tl_trg": {
        "test": "https://raw.githubusercontent.com/UniversalDependencies/UD_Tagalog-TRG/master/tl_trg-ud-test.conllu",
    },
    "tl_ugnayan": {
        "test": "https://raw.githubusercontent.com/UniversalDependencies/UD_Tagalog-Ugnayan/master/tl_ugnayan-ud-test.conllu",
    },
```